data_discovery_ai/utils/es_connector.py

```diff
@@ -43,32 +43,72 @@ def connect_es(config: configparser.ConfigParser) -> Elasticsearch:
 
 def search_es(client: Elasticsearch):
     index = "es-indexer-staging"
-    start_index = 0
     dataframes = []
     batch_size = 100
-    resp = client.search(index=index, body={"query": {"match_all": {}}})
-    total_number = resp["hits"]["total"]["value"]
 
+    # get document count
+    count_resp = client.count(index=index)
+    total_number = count_resp["count"]
     rounds = (total_number + batch_size - 1) // batch_size
 
-    # TODO: refactor the search query method.
-    for round in tqdm(range(rounds), desc="searching elasticsearch"):
-        search_query = {
-            "size": batch_size,
-            "from": start_index,
+    # get the first pit
+    first_pit_resp = client.open_point_in_time(
+        index=index,
+        keep_alive="1m",
+    )
+    first_pit = first_pit_resp["id"]
+
+    # the first search
+    first_query_body = {
+        "size": 100,
```

Comment:
hard-coded value `100`? why has to be `100`? and should it be declared in a global config/constant file?